lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -115,6 +116,16 @@ export default function ContentSelector({
         />
       );
       break;
+    case 'blackboardLmsFile':
```

Comment:
Since it is not possible to be using Canvas files and Blackboard files at the same time, we might be able to combine the `lmsFile` and `blackboardLmsFile` cases here and just switch the `listFilesApi` prop and button caption depending on the LMS.